src/main/java/com/google/cloud/spanner/pgadapter/statements/DdlExecutor.java

```diff
@@ -66,14 +73,52 @@ static String unquoteIdentifier(String identifier) {
    * make the statement compatible with Cloud Spanner.
    */
   StatementResult execute(ParsedStatement parsedStatement, Statement statement) {
+    Tuple<ParsedStatement, Statement> replaced = replace(parsedStatement, statement);
+    parsedStatement = replaced.x();
+    statement = replaced.y();
+
+    if (!backendConnection.getSessionState().isSupportDropCascade()) {
+      return connection.execute(translate(parsedStatement, statement));
+    } else {
+      ImmutableList<Statement> allStatements =
+          getDependentStatements(translate(parsedStatement, statement));
+      if (allStatements.size() == 1) {
+        return connection.execute(allStatements.get(0));
+      } else {
+        boolean startedBatch = false;
+        try {
+          if (!connection.isDdlBatchActive()) {
+            connection.execute(Statement.of("START BATCH DDL"));
+            startedBatch = true;
+          }
+          StatementResult result = null;
+          for (Statement dropDependency : allStatements) {
+            result = connection.execute(dropDependency);
```

Comment:
Hmm... We don't need to validate it, because we know that it is a `StatementResult.NO_RESULT` in all cases, as the statements are all `DROP` statements that are generated in this class. The reason that it was assigned here, was to guarantee that there is a result that we can return, but that is also not really necessary. Instead, we can just return the static field `NO_RESULT` that we already have.